docker/Dockerfile

```diff
@@ -113,18 +114,6 @@ RUN echo "## Update and install packages" \
     && rm -rf /usr/share/man/??_* \
     && echo "## Done"
 
-# ------------------------------------------------------------------------------
-# Internal test suite of tools that this image provides
-COPY test_dockerimage.sh /emsdk/
-
-RUN echo "## Internal Testing of image" \
-    && /emsdk/test_dockerimage.sh \
-    && echo "## Done"
-
-# ------------------------------------------------------------------------------
-# Copy this Dockerimage into image, so that it will be possible to recreate it later
-COPY Dockerfile /emsdk/dockerfiles/emscripten-core/emsdk/
```

Comment:
Ditto, what is common practice here?

Reply:
I haven't seen it very often, but I think it might be useful for reproducible builds. I removed it because the `Dockerfile` already exists in `/emsdk/docker` (assuming this is the same as the current build `Dockerfile`).

Reply:
Great lgtm

Reply:
That was the goal also. Again it comes from early development of image, where I wanted to reproduce 1:1 images that were published to the web. I don't think it's needed currently.